README.md

```diff
@@ -0,0 +1,159 @@
+- Reverse, takes a string, and returns it reversed.
+- Fizzbuzz takes an integer and returns some text when conditions are accomplished.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Factorial, takes an integer value and returns the factorial.
+- Reverse, takes a string, and returns it reversed.
+- Fizzbuzz takes an integer and returns some text when conditions are accomplished.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running.
+- You should be able to use [IRB](<https://en.wikipedia.org/wiki/Ruby_(programming_language)#Features>).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:lmunozm1702/tdd-ruby.git
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  ruby solver.rb
+```
+
+To test the project, execute the following command:
+
+```
+  rspec spec/solver_spec.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abeera Tahir**
+
+- GitHub: [@AbeeraTahir](https://github.com/AbeeraTahir)
+- Twitter: [@AbeeraTahir8](https://twitter.com/AbeeraTahir8?t=z5CjMpmHMZmS98i09gUpYA&s=08)
+- LinkedIn: [Abeera Tahir](https://www.linkedin.com/in/abeera-tahir/)
+
+👤 **Luis Muñoz**
+
+- GitHub: [@lmunozm1702](https://github.com/lmunozm1702)
+- Email: [l.munoz.m@outlook.com](mailto:l.munoz.m@outlook.com)
+- Twitter: [@lmunozm1702](www.twitter.com/lmunozm1702)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project and want to support me make cooler projects Give this project a Star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job team 🤝

- [ ] [Optional] You may fix the broken MIT license link to make your readme file more professional.